pages/common/img2pdf.md

```diff
@@ -0,0 +1,32 @@
+# img2pdf
+
+> Command-line lossless converter of raster images to PDF.
+> More information: <https://gitlab.mister-muffin.de/josch/img2pdf>.
+
+- Convert images to PDF:
+
+`img2pdf {{path/to/image1.jpg}} {{path/to/image2.jpg}} --output {{path/to/file.pdf}}`
+
+- Convert all PNGs in the current directory to a single pdf:
+
+`img2pdf {{*.png}} --output {{path/to/file.pdf}}`
+
+- Convert only the first page of multi-page images to pdf:
+
+`img2pdf {{path/to/file.gif}} --first-frame-only --output {{path/to/file.pdf}}`
+
+- Auto orient to A4 page size in landscape mode and a border of 2cm horizontally and 5.1cm vertically:
+
+`img2pdf {{path/to/file.jpg}} --auto-orient --pagesize {{A4^T}} --border {{2cm:5.1cm}} --output {{path/to/file.pdf}}`
+
+- Shrink only larger images to a 10cm by 15cm rectangle inside a 30x20cm page:
+
+`img2pdf {{path/to/file.tiff}} --pagesize {{30cmx20cm}} --imgsize {{10cmx15cm}} --fit {{shrink}} --output {{path/to/file.pdf}}`
+
+- Convert an image to PDF including specific metadata:
```

Comment:
Note that images can contain metadata in the form of EXIF/XMP tags, so it' worth differentiating here. Perhaps this will read better?
```suggestion
- Convert an image to PDF, and specify metadata for the resulting file:
```